tutorials/basics_tutorial.ipynb

```diff
@@ -0,0 +1,1500 @@
+{
```

Comment:
<p>Are we leaving this heading in? Perhaps we can remove if making a separate visualisation tutorial in #40</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MDAnalysis/solvation-analysis/pull/36/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>